Conceptpower+Spring/src/integration-test/java/edu/asu/conceptpower/rest/ConceptSearchIT.java

```diff
@@ -60,10 +176,32 @@ public void test_searchConcept_searchWithWordAndPosAndSimilarToInXml() throws Ex
                 this.getClass().getClassLoader().getResourceAsStream("output/conceptWithWordPosAndSimilarTo.xml"));
 
         this.mockMvc
-                .perform(MockMvcRequestBuilders.get("/ConceptSearch").param("word", "Douglas").param("pos", "noun")
-                        .param("similar_to", "http://viaf.org/viaf/248802520")
+                .perform(MockMvcRequestBuilders.get("/ConceptSearch").param("word", "Douglas Weiner")
+                        .param("pos", "noun").param("similar_to", "http://viaf.org/viaf/248802520")
+                        .param("operator", SearchParamters.OP_AND).accept(MediaType.APPLICATION_XML_VALUE))
```

Comment:
you don't have to do it everywhere, but in one case it would make sense to test for the default operator.